### DISCUSSION TITLE
Creating tiles on the fly

### DISCUSSION THREAD

**Discussion by MicahDavid** [react-native-maps/react-native-maps#4991] [Q&A]
I"m looking for a way to create tile images on the fly. On the web I use Leaflet api, and generate a canvas or svg image on the fly for the tile images. I know Google Maps has similar functionality. Is there any ability for react native maps to create tiles on the fly based on tile coordinates?

**Comment by Dosbodoke** [comment]
Any updated on this?

**Reply by MicahDavid** [reply]
@Dosbodoke I haven't looked into this problem for a while, but have also not encountered a solution. Please inform if you happen to find a good solution. Thanks.
